src/component/App/App.js

```diff
@@ -0,0 +1,14 @@
+import './App.css';
+import ButtonPanel from '../buttonPanel';
+import Display from '../Display';
+
+function App() {
+  return (
+    <div className="App">
+      <ButtonPanel />
+      <Display />
+    </div>
```

Comment:
- [ ] Kindly wrap your children components in a React.fragment as required.
![image](https://user-images.githubusercontent.com/57832450/129207231-c0baeef2-277c-4749-a8c7-3fd3ebc37c69.png)

- [ ] Kindly place the` Display` component above the `ButtonPanel `component to improve user experience.